configure

```diff
@@ -542,7 +545,16 @@ def is_arm_hard_float_abi():
 def host_arch_cc():
   """Host architecture check using the CC command."""
 
-  k = cc_macros()
+  # use 'cc', as CC may be set to a target arch compiler command
+  # in case of cross-compilation
+  if sys.platform.startswith('aix'):
+    k = cc_macros('gcc')
+    matchup = {
+      '_ARCH_PPC'   : 'ppc',
+      '_ARCH_PPC64' : 'ppc64',
+    }
+  else:
+    k = cc_macros('cc')
```

Comment:
`k = cc_macros()`?  Passing 'cc' is a subtle change in behavior on non-AIX.

I don't get why you need to special-case here for cross-compilation.  If you're cross-compiling, you normally want to know the capabilities of the target platform, not the host platform.